Sources/SwiftParser/Lexer.swift

```diff
@@ -1618,7 +1618,13 @@ extension Lexer.Cursor {
     self.advance(while: { $0.isValidIdentifierContinuationCodePoint })
 
     let text = tokStart.textUpTo(self)
-    return (RawTokenKind(keyword: text) ?? .identifier, [])
+    if let keywordKind = RawTokenKind(keyword: text) {
+      return (keywordKind, [])
+    } else if let keyword = Keyword(text), keyword.isLexerClassified {
+      return (.keyword(keyword), [])
+    } else {
+      return (.identifier, [])
+    }
```

Comment:
Given how often this is called, IMO we should try keep it as optimal as possible. Ie. `public init?(keyword text: SyntaxText)` should keep the `isLexerClassified` initializations. Seems like we could probably just make that initializer non-nil as well + return `.identifer`.

Reply:
That may also mean there's no need for the `Keyword` `SyntaxText` initializer, though I haven't checked where else it is called.

Reply:
My hope was that we could get rid of the remaining pound keywords and remove the `RawTokenKind(keyword: SyntaxText)` initializer entirely.